masterlist.yaml

```diff
@@ -4035,6 +4057,10 @@ plugins:
     clean:
       - crc: 0xAD717340 # v4.1
         util: SSEEdit v3.2.2
+  - name: 'DBM_OCS_Patch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/11802/'
+        name: 'Legacy of the Dragonborn Patches SE on Nexus Mods'
```

Comment:
Generally there should be some other information besides name and URL to warrant adding an entry.

Reply:
I just checked SSEEdit and no tags were recommended so that can be deleted then.  I'll keep that in mind moving forward, thank you for the feedback!

Reply:
I'll be adding to it, so it can stay.